src/components/App.js

```diff
@@ -5,11 +5,31 @@ import ButtonPanel from './ButtonPanel';
 // eslint-disable-next-line no-unused-vars
```

Comment:
- do not modify eslint, kindly do the same in other areas where you disabled the eslint

Reply:
Done